src/main/java/org/opentripplanner/api/model/ApiTripSearchMetadata.java

```diff
@@ -9,12 +9,18 @@
      * This is the time window used by the raptor search. The window is an optional parameter and
      * OTP might override it/dynamically assign a new value.
      * <p>
+     * Note! This parameter is NOT adjusted when the number of itineraries are reduced due to the
+     * request {@code numOfItineraries} parameter. When getting the next results
+     * ({@code arriveBy=true}) or previous results({@code arriveBy=false}), be sure to set the
+     * {@code numOfItineraries} high enough to include all itineraries. If not, the combined results
+     * will be missing trips.
+     * <p>
```

Comment:
We might want to make it more explicit, that this is only when changing the direction of the search. Now it is not clear, when skimming the text. When searching in the same direction, there is no need to se the `numOfItineraries` any higher.

Reply:
I tried to make it clearer se the new doc on: `searchWindow`, `nextDateTime` and `prevDateTime`.